src/tests/compiler/class/methodDeclarationTests.ts

```diff
@@ -109,6 +111,23 @@ describe(nameof(MethodDeclaration), () => {
         });
     });
 
+    describe(nameof<MethodDeclaration>(m => m.isOverload), () => {
+        function doTest(startingCode: string, methodName: string, expected: boolean) {
+            const {firstChild, sourceFile} = getInfoFromText<ClassDeclaration>(startingCode);
+            const method = firstChild.getInstanceMethods().find(m => m.getName() === methodName);
+            expect(method).to.not.be.undefined;
+            expect(method!.isOverload()).to.be.equals(expected);
+        }
+
+        xit("should be false for a non overloaded abstract method ?", () => {
+            doTest(`
```

Comment:
We'll have to add tests for other scenarios for this. Perhaps as part of a different pull request though.